lib/rps.rb

```diff
@@ -0,0 +1,25 @@
+class RPS
+
+  attr_reader :player, :opponent
+
+  def initialize(player, opponent)
+    @player = player.downcase
+    @opponent = opponent
+  end
+
+  def result
+    return "draw" if draw? == true
+    player_wins? == true ? "win" : "lose"
+  end
+
+  private
+
+  def player_wins?
+    winning_round = [["rock", "scissors"], ["scissors", "paper"], ["paper", "rock"]]
```

Comment:
Really smart how you created the logic for the winning round - I was trying to wrap my head around how best to do this and this really clear.